src/timer/timer_game_realtime.cpp

```diff
@@ -53,12 +53,18 @@ void TimeoutTimer<TimerGameRealtime>::Elapsed(TimerGameRealtime::TElapsed delta)
 	}
 }
 
+/**
+ * Every-tick handler for real-time timers.
+ * @return Always false, since this value is required by the template but not used for anything by this timer.
```

Comment:
This reasoning is very hard to parse. Why not keep it simple, and just say, as general rule for all `Elapsed`: `true iff the time progressed`, and that means this function would also just return true. Easier to explain, less vague statements that only makes sense when you know what you were struggling with that the time of writing :)